src/pages/editor/EditorDoi.tsx

```diff
@@ -17,43 +17,58 @@ export const EditorDoi = () => {
       ) : (
         <>
           {!customer.doiAgent.username ? (
-            <Typography paragraph>
+            <Typography paragraph fontWeight="bold">
               <Trans t={t} i18nKey="editor.doi.institution_cannot_create_doi">
                 <MuiLink href={'mailto:kontakt@sikt.no'} target="_blank" rel="noopener noreferrer" />
               </Trans>
             </Typography>
           ) : (
-            <Typography paragraph>{t('editor.doi.institution_can_create_doi')}</Typography>
+            <Typography paragraph fontWeight="bold">
+              {t('editor.doi.institution_can_create_doi')}
+            </Typography>
           )}
 
           <Box
             sx={{
-              bgcolor: 'grey.400',
-              width: 'fit-content',
-              p: '1rem 2rem',
               display: 'flex',
-              gap: '3rem',
-              'h3, p': {
-                fontSize: '1rem',
-                fontWeight: 500,
-              },
+              columnGap: '1rem',
+              width: '100%',
+              'h3, p': { fontWeight: '500' },
             }}>
-            <div>
-              <Typography variant="h3">{t('basic_data.institutions.doi_repo_id')}</Typography>
```

Comment:
`basic_data.institutions.doi_repo_id` kan fjernes fra nbTranslations.json

Reply:
Ser begge nøklene jeg forslår å fjerne brukes andre plasser. Men de som er fjernet og de som er lagt til skal kanskje merges til å bli to keys i stedet for 4? Ser ut som de formidler samme greia på ulik måte atm 🤔

Reply:
Ja, tenkte litt på det, men den ene er en kolonne-header i en tabell (hvor det raskt blir trangt med enda et ord), og den andre er på en plass jeg ikke var sikker på om forandringen er gyldig. Skal høre med PO om sistnevnte også kan byttes.

Reply:
Bra du spør. Den sistnevnte skal ikke endres i det hele tatt. Feil i skissene.